### PR TITLE
Add weather data

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
See #66

File: app/models/incident_report.py [hunk 1]
```diff
@@ -58,6 +60,9 @@ def __init__(self, notify_workers_upon_creation=True, **kwargs):
         if self.date is None:
             self.date = datetime.now()
 
+        if self.weather is None and self.location is not None:
+            self.weather = get_current_weather(self.location)
+
```
Comment (rileynwong) [inline app/models/incident_report.py:65]: might want to add an else statement that'll set `self.weather = ''`

File: app/templates/reports/manage_report.html [hunk 2]
```diff
@@ -42,6 +42,13 @@
         <tr><td>Time</td><td>{{ report.date.strftime('%I:%M %p') }}</td></tr>
         <tr><td>Duration (h:m:s)</td><td>{{ report.duration}}</td></tr>
         <tr><td>Agency</td><td>{{ report.agency.name }}</td></tr>
+        <tr><td>Weather conditions during incident</td>
+            <td>
+                {% for line in report.weather.splitlines() %}
```
Comment (rileynwong) [inline app/templates/reports/manage_report.html]: if `weather == None`, will this error out?

Comment (sandlerben) [inline app/templates/reports/manage_report.html]: good call

**Comment by sandlerben** [comment]
@nancywong look good now?

**Comment by rileynwong** [comment]
yeah lgtm

**Pull request merged**